san/v2_metrics_list.py

```diff
@@ -73,5 +73,87 @@
       "withdrawal_transactions",
       "supply_on_exchanges",
       "percent_of_total_supply_on_exchanges",
-      "amount_in_top_holders"
+      "amount_in_top_holders",
```

Comment:
can you remind me why we list these manually instead of fetching from the api like this ?
```
{
  getAvailableMetrics
}
```

Reply:
If I remember correctly, it had something to do with the tests and the fact that you can't mock 2 responses in one test, otherwise it worked